src/components/Select/Select.module.scss

```diff
@@ -0,0 +1,104 @@
+.root {
+  --remove: #d6d6d6;
+  --remove-hover: #c6c6c6;
```

Comment:
very nitpicky but i would like a broader variable name for these as they are global.